### ISSUE TITLE
https://cloud-span.github.io/cloud-admin-guide-1-setting-work-environments/

### ISSUE THREAD

**Issue by 3mmaRand** [Cloud-SPAN/cloud-admin-guide-0-overview#2]
Possible edits:
Delete: 

> We have not tested the Scripts with other shells, or on MacOS or Windows using the Git Bash program. The course does not cover such compatibility or configuration issues.

Add this info (from a later page) here too:

> You Linux machine can be any of the following:
> 
> a Linux laptop or desktop machine.
> a Linux virtual machine running within VMware (or Virtual Box, etc.) — and VMware (Virtual Box, etc.) running on a Windows, Linux or MacOS laptop or desktop machine.
> a remote Linux server (e.g., an AWS Linux instance) accessed with ssh.
> Ubuntu Linux within the Windows Subsytem for Linux (WSL) running on a Windows laptop or desktop machine.

**Comment by JorgeBCh** [comment]
Yes that part is going to change somewhat. Particularly,  right now I think it no longer makes sense to talk about a "Linux environment" but rather about a "Bash Terminal /Shell environment". 
THank you very much for looking into this as this is the kind of things that I might take a biased decision due to my background. We can talk about it today in our meeting. Thanks.
